include/onnxruntime/core/providers/cuda/cuda_context.h

```diff
@@ -28,38 +28,47 @@
   cudnnHandle_t cudnn_handle = {};
   cublasHandle_t cublas_handle = {};
   OrtAllocator* deferred_cpu_allocator = {};
+  // below are cuda ep options
+  int16_t device_id = 0;
+  int32_t arena_extend_strategy = 0;
+  int32_t cudnn_conv_algo_search = 0;
+  bool cudnn_conv_use_max_workspace = true;
+  bool enable_cuda_graph = false;
```

Comment:
>bool enable_cuda_graph = false; [](http://example.com/codeflow?start=2&length=31)

i don't think custom op developer need to aware whether cuda graph is used. could we also remove this one?